app/controllers/v0/example_controller.rb

```diff
@@ -0,0 +1,21 @@
+# example controller to show use of logging in with sesssions controller
+
+module V0
+  class ExampleController < ApplicationController
+
+    def index
+      render json: { "message": "Welcome to the vets.gov API"}
+    end
+
+    # requires log in
+    def welcome
+      if session[:user]
```

Comment:
Is this conditional for demo purposes? Could it become a devise like before_action?